core/integration/file_test.go

```diff
@@ -393,6 +393,26 @@ func (FileSuite) TestContents(ctx context.Context, t *testctx.T) {
 	}
 }
 
+func (FileSuite) TestDigest(ctx context.Context, t *testctx.T) {
```

Comment:
Can we expand these tests? Ideally each test should have have two files with the same contents, but different metadata.

Then in "without metadata", the files will have the same digest, but in "with metadata" they'll be different.

Reply:
Done :D